File: DriftGame/uracer-desktop/src/com/bitfire/uracer/game/logic/gametasks/trackeffects/TrackEffectType.java
package com.bitfire.uracer.game.logic.gametasks.trackeffects;

/**
 * Defines the type of special effect, it also describes their rendering order (FIXME)
 */
public enum TrackEffectType {
    CarSkidMarks(1),
    CarSmokeTrails(2);

    public final int id;

    TrackEffectType(int id) {
        this.id = id;
    }
}

File: DriftGame/uracer-desktop/src/com/bitfire/uracer/game/logic/gametasks/trackeffects/TrackEffectType.kt
package com.bitfire.uracer.game.logic.gametasks.trackeffects

/**
 * Defines the type of special effect, it also describes their rendering order.
 */
enum class TrackEffectType(@JvmField val id: Int) {
    CarSkidMarks(1),
    CarSmokeTrails(2)
}
